code/game/machinery/computer/abnormality_work.dm

```diff
@@ -53,6 +53,8 @@
 	dat += "<b><span style='color: [THREAT_TO_COLOR[datum_reference.threat_level]]'>\[[THREAT_TO_NAME[datum_reference.threat_level]]\]</span> [datum_reference.name]</b><br>"
 	if(datum_reference.overload_chance != 0)
 		dat += "<span style='color: [COLOR_VERY_SOFT_YELLOW]'>Current success chance is modified by [datum_reference.overload_chance]%</span><br>"
+	if(datum_reference.understanding != 0)
```

Comment:
Same here, it'd be a good idea to use >

Reply:
Again, physically can't be below 0 or above max understanding as they are clamped between 0 and max_understanding

Reply:
`understanding = clamp((understanding + (max_understanding/20)), 0, max_understanding)`

Reply:
This also specifically has it so that if the Abno hasn't been worked on, the menu doesn't show at all, just like Overload chance